easybuild/framework/easyconfig/tools.py

```diff
@@ -282,7 +287,7 @@ def det_easyconfig_paths(orig_paths, from_pr=None, easyconfigs_pkg_paths=None):
                     break
 
                 # ignore subdirs specified to be ignored by replacing items in dirnames list used by os.walk
-                dirnames[:] = [d for d in dirnames if not d in build_option('ignore_dirs')]
+                dirnames[:] = [d for d in dirnames if d not in build_option('ignore_dirs')]
```

Comment:
@fgeorgatos: I think it's about interpreting the code. `not x in xs` could be interpreted as `not (x in xs)` or `(not x) in xs`. You don't have that problem with `x not in xs`.